engine/servermaster/server.go

```diff
@@ -68,31 +71,19 @@ import (
 const (
 	leaderElectionTable  = "leader_election"
 	resignLeaderDuration = 10 * time.Second
+	grpcConnBufSize      = 32 * 1024
 )
 
-// use a slice instead of map because in small data size, slice search is faster
-// than map search.
-var masterRPCLimiterAllowList = []string{
```

Comment:
This log filter seems missing, do we need to add a log middleware

Reply:
We have log verbose request/response in error middleware.

Reply:
The error middleware records
- verbose request if error happens
- debug log for normal request/response
But it lacks info logs for some critical method (without error), such as `CreateJob`, `CancelJob` etc.

Reply:
Oh, I missed it. I will add it back.

Reply:
Added in https://github.com/pingcap/tiflow/pull/7703/commits/4830331e5d84e5bd847cb744148824f06c534190.